README.md

```diff
@@ -32,28 +32,30 @@ Winit is designed to be a low-level brick in a hierarchy of libraries. Consequen
 show something on the window you need to use the platform-specific getters provided by winit, or
 another library.
 
-```rust
+```rust, ignore
 use winit::{
     event::{Event, WindowEvent},
-    event_loop::{ControlFlow, EventLoop},
+    event_loop::EventLoop,
     window::WindowBuilder,
 };
 
-fn main() {
-    let event_loop = EventLoop::new();
-    let window = WindowBuilder::new().build(&event_loop).unwrap();
-
-    event_loop.run(move |event, elwt| {
-        elwt.set_control_flow(ControlFlow::Wait);
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let event_loop = EventLoop::new().unwrap();
+    let window = WindowBuilder::new().build(&event_loop)?;
 
-        match event {
+    event_loop
+        .run(move |event, _elwt, cf| match event {
+            Event::RedrawRequested(_) => {
+                println!("Redraw requested");
+            }
```

Comment:
What's the reason to revert this part? It was correct before, the `run` closure has only 2 arguments passed and it was all correct. I think the only issue is the `EventLoop::new()` missing a change to call `?`.

Reply:
Hm, maybe we should remove the docs then entirely. The README could only reflect the branch it's on, otherwise it's impossible to validate the docs in the first place. We do have examples and the doc comment in the source code in lib.rs.

Tracking the example manually won't work, since it already failed. And any automated testing could be done only for example which we compile.